source/clear-linux/tutorials/kubernetes.rst

```diff
@@ -211,54 +164,61 @@ If you choose the `flannel` add-on, then you must add the following to the
 
     --pod-network-cidr 10.244.0.0/16
 
-If you are using CRI-O and `flannel` and you want to use Kata Containers, edit
-the :file:`/etc/crio/crio.conf` file to add:
+If you are using CRI-O and `flannel` and you want to use Kata Containers, edit the :file:`/etc/crio/crio.conf` file to add:
 
 ..  code-block:: bash
 
     [crio.runtime]
     manage_network_ns_lifecycle = true
 
-**Notes about Weave Net add-on**
+.. TODO: Should Weave and Calico be mentioned below, or is Flannel enought?
```

Comment:
might be  good to tell the user that they can use any networkk plugin they choose, just that we have not tested with all of them and if they do find issues they can report a bug in our github? i am trying to avoid giving them the impression that kata will only work with flannel.

Reply:
I will add a note:

You may choose to user other network plugins; however, they are not validated in the scope of this document.